File: src/main/java/net/igneo/icv/entity/voidSpike/VoidSpikeRenderer.java
package net.igneo.icv.entity.voidSpike;

import net.minecraft.client.renderer.entity.EntityRendererProvider;
import software.bernie.geckolib.renderer.GeoEntityRenderer;

public class VoidSpikeRenderer extends GeoEntityRenderer<VoidSpikeEntity> {
    public VoidSpikeRenderer(EntityRendererProvider.Context renderManager) {
        super(renderManager, new VoidSpikeModel());
    }
}

File: src/main/java/net/igneo/icv/entity/leggings/voidSpike/VoidSpikeRenderer.kt
package net.igneo.icv.entity.leggings.voidSpike

import net.minecraft.client.renderer.entity.EntityRendererProvider
import software.bernie.geckolib.renderer.GeoEntityRenderer

class VoidSpikeRenderer(renderManager: EntityRendererProvider.Context) :
    GeoEntityRenderer<VoidSpikeEntity?>(renderManager, VoidSpikeModel())
